blockchain/blockchain-wallet-v4-frontend
# Feature/misc bugs6

## Description
- Various bug fixes
- Implementation notifications settings

## Change Type
Please enter one or more of the following: 
- Bug Fix
- Feature

## Testing Steps
Detail the steps required for the reviewer(s) to verify and test these changes.
- Test if notifications can be toggled, persist after login again between v3/v4, and that you receive those notifications by sms/email.

## Code Checklist
